modules/interactive_music/audio_stream_interactive.cpp

```diff
@@ -881,12 +881,19 @@ void AudioStreamPlaybackInteractive::_mix_internal(int p_frames) {
 		mix_buffer[i] = AudioFrame(0, 0);
 	}
 
+	bool any_active = false;
 	for (int i = 0; i < stream->clip_count; i++) {
 		if (!states[i].active) {
 			continue;
 		}
 
 		_mix_internal_state(i, p_frames);
+
+		any_active |= states[i].active;
```

Comment:
Small nitpick, I would keep `|=` for bitwise operations, and use `||` for logical operations.
```suggestion
		any_active = states[i].active || any_active;
```